src/client/src/commons/menu/editor/menuEditorForm.js

```diff
@@ -195,8 +201,64 @@ class MenuEditorForm extends React.Component {
                 <TranslationText firstUpperCase id="stratigraphy" />
               </List.Header>
             </List.Content>
+            <div style={{ marginLeft: "2em" }}>
+              {!this.state.stratigraphyIsVisible && (
+                <List.Icon
+                  name="angle down"
+                  size="big"
+                  verticalAlign="middle"
+                />
+              )}
+              {this.state.stratigraphyIsVisible && (
+                <List.Icon name="angle up" size="big" verticalAlign="middle" />
+              )}
+            </div>
           </List.Item>
-          {/* Completion */}
+          {this.state.stratigraphyIsVisible && (
+            <>
+              <List.Item
+                active={
+                  location.pathname ===
+                  process.env.PUBLIC_URL +
+                    "/editor/" +
+                    match.params.id +
+                    "/stratigraphy/chronostratigraphy"
+                }
+                onClick={() => {
+                  history.push(
+                    process.env.PUBLIC_URL +
+                      "/editor/" +
+                      match.params.id +
+                      "/stratigraphy/chronostratigraphy",
+                  );
+                }}
+                style={{
+                  padding: "1em",
+                  paddingLeft: 40,
+                  display: "flex",
+                  borderLeft:
+                    location.pathname ===
+                    process.env.PUBLIC_URL +
+                      "/editor/" +
+                      match.params.id +
+                      "/completion/casing"
```

Comment:
Q: Wären die `pathname`-Vergleiche einfacher lesbar, wenn wir hier mit String-Interpolation arbeiten?

Reply:
Wird in https://github.com/geoadmin/suite-bdms/issues/486 ausprobiert.